docs/architecture/grpc-for-wcf-developers/why-grpc.md

```diff
@@ -44,6 +44,16 @@ gRPC allows clients to specify a maximum time for an RPC to finish. If the speci
 
 gRPC is implicitly secure when it's using HTTP/2 over a TLS end-to-end encrypted connection. Support for client certificate authentication (see [chapter 6](security.md)) further increases security and trust between client and server.
 
+## Comparison with CoreWCF
+
+A notable alternative to gRPC for replacing WCF services when migrating to .NET is [CoreWCF](https://github.com/corewcf/corewcf). Both gRPC and CoreWCF are Microsoft endorsed paths forward for WCF applications and each comes with its own benefits and drawbacks.
+
+CoreWCF is a community-owned .NET Foundation project supported by Microsoft that implements many WCF server APIs for .NET. Although CoreWCF is now adding new modern features, it began as an effort to allow existing WCF services to work with minimal changes on .NET. Patterns for starting WCF services are different in CoreWCF and not all configuration options are available (some configuration must now be done in code), but most other WCF APIs are available in CoreWCF. Services and interfaces can often migrate with few changes. Because of this, a key benefit of CoreWCF is its very similar surface area and programming model compared to WCF.
+
+On the other hand, gRPC is a modern remote communication solution with a number of features, as discussed previously. Benefits of using gRPC include its interoperability across languages, its relatively simple modern API, and a broad community ecosystem.
```

Comment:
SOAP is intended to be interoperable across languages, and generally the Basic Profile is. It's when you start using more of the SOAP spec that things start getting a little trickier. The original sentence implied there isn't interoperable implementations for WCF, whereas it's actually more consistent (and more languages/platforms) interoperable support for gRPC so I think the "better" qualifier is more accurate. I'm happy if you want to use a qualifier other than better, the key thing is gRPC is more broadly supported cross language than SOAP rather than SOAP not being supported cross language.  
```suggestion
On the other hand, gRPC is a modern remote communication solution with a number of features, as discussed previously. Benefits of using gRPC include better interoperability across languages, its relatively simple modern API, and a broad community ecosystem.
```